specs/meroxa/resources_create_bigquery.yaml

```diff
@@ -0,0 +1,46 @@
+---
+# The name of the workflow.
+name: Add a Google BigQuery resource to the Meroxa Platform
+# The corresponding command for the workflow. Any arguments should be surrounded with two curly braces. E.g `command {{arg}}`.
+command: |-
+    meroxa resources create {{resource_name}} --type bigquery --url "bigquery://{{gcp_project_id}}/{{gcp_dataset_name}}" --client-key "$(cat {{gcp_service_account_json_file}})"
+# Any tags that the workflow should be categorized with.
+tags:
+  - meroxa
+  - data
+  - realtime
+  - bigquery
```

Comment:
small nit: We tend to opt for less tags in workflows because we find that this clutters the workflows UI in Warp. Could we just use the `meroxa` tag here and below instead?

Reply:
@alokedesai that works!